autoscale-dockerswarm-container/pom.xml

```diff
@@ -40,43 +40,52 @@
         <dependency>
             <groupId>com.github.cafapi</groupId>
             <artifactId>caf-api</artifactId>
+             <scope>runtime</scope>
         </dependency>
         <dependency>
             <groupId>com.github.cafapi.config</groupId>
             <artifactId>${caf.config.impl}</artifactId>
+            <scope>runtime</scope>
```

Comment:
I don't think we should bother with these `<scope>runtime</scope>` declarations in container projects.  If we are going to put them in then we should do it consistently, i.e. we should also be adding them to the `autoscale-marathon-container` project.

Reply:
Added <scope>runtime</scope> for autoscale-marathon-container project.